### ISSUE TITLE
op23_no01: does not render when `!!!filter: flipper -a` is used

### ISSUE THREAD

**Issue by bel28kent** [bel28kent/Mysterium#47]
Without `!!!filter: flipper -a`:

<img width="1440" alt="Screen Shot 2024-04-22 at 3 53 36 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/59af9575-e092-4230-8489-49d7f51943cf">

With `!!!filter: flipper -a`:

<img width="1440" alt="Screen Shot 2024-04-22 at 3 54 43 PM" src="https://github.com/bel28kent/Mysterium/assets/81043032/2cd4dd47-84da-4353-9c8a-92b6736fb655">

[kern](https://github.com/bel28kent/Mysterium/blob/22-phase21-subspine-issue/op23/no_stems-scriabin-op23_no01.krn)

**Comment by craigsapp** [comment]
On lines:

<img width="674" alt="Screenshot 2024-04-22 at 1 01 18 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/09af103a-b836-4fba-bbb0-ceb052daadf2">

Spine merging cannot be done within a measure with `flipper` related to a limitation in how conversion to MEI/MusicXML is done.  These formats use the concept of voice/layers instead of subspines, so I read the subspines from left to right to assign to voices/layers.  In this case the second layer/voice music is falling into the first layer when flipping, so the first voice becomes overfilled with notes since it is stealing the rest from the end of layer 2.   When not flipping, there is no problem in this case since the first layer is not disrupted and the second layer disappears at the end of the measure.  When flipper is used, both forms of the data cannot have merges that end in the middle of a measure (but a split can happen in the middle of a measure).  

Something might be able to be done to fix this limitation, but it would take me a while to think about it (I would keep track of the subspines and pre-assign voice/track numbers to them, but a complexity occurs when they merge for these assignments).

For now, it is best to add spacer rests to prevent merging in the middle of a measure:

<img width="962" alt="Screenshot 2024-04-22 at 1 10 15 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/570df0aa-058e-4765-9560-a3f9c1235f09">

In general I keep the split spines lasting longer if there is just a brief merger otherwise:

<img width="631" alt="Screenshot 2024-04-22 at 1 12 33 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/7011487d-38ec-41f6-ba2c-0e1f841cb2de">

Depending on analysis intentions, the following encoding may be more suiltable:

<img width="630" alt="Screenshot 2024-04-22 at 1 14 18 PM" src="https://github.com/bel28kent/Mysterium/assets/3487289/d1315ff1-0de0-4da6-8a06-6aef2ec10b69">

For example, if you want to explicitly say that the second voice is continuous as a separate melodic stream, as compared to a set of melodic fragments in the original encoding, then this encoding arrangement is better.   Also, the rests that are generating the merges can be thought of as two over-struck rests, i.e., there are two rests but they overlap each other 100%.

---------------

Somewhat related: I will create a `*rjoin` interpretation (sooner or later).   This would allow both rest in two subspines that should only be displayed once in the notation (rather than two separate rests):

```tsv
16CC# 16C#	16r	16r	.
```

It would go something like this:

```tsv
*	*rjoin	*rjoin	*
16CC# 16C#	16r	16r	.
```

This would be persistent until `*Xrjoin` is encountered.

This is related to automatic rest placement.  I used to place rests in my Humdrum-to-MEI converter, but since last month I have removed that feature and let verovio automatically place rests vertically, since it is better at doing that than previously.  Verovio vertically adjusts rests in two layers automatically to separate them, while my default was to merge them into a single rest by default.   The `*rjoin` interpretation will allow easy switching between merged and separated rests style, and allow verovio to still manage vertical adjustments of rest against notes (which then makes it less necessary to deal with in Humdrum or add automatically within the Humdrum-to-MEI converter.
